core/trino-main/src/main/java/io/trino/execution/CreateTableTask.java

```diff
@@ -132,7 +132,7 @@ ListenableFuture<Void> internalExecute(CreateTable statement, Session session, L
         }
         catch (TrinoException e) {
             if (e.getErrorCode().equals(UNSUPPORTED_TABLE_TYPE.toErrorCode())) {
-                throw semanticException(TABLE_ALREADY_EXISTS, statement, "Table '%s' of unsupported type already exists", tableName);
+                throw new TrinoException(UNSUPPORTED_TABLE_TYPE, format("The type of table '%s' is unsupported", tableName));
```

Comment:
The original exception seems better. What's the motivation of this change?

Reply:
Because I thought the exception should not be `TABLE_ALREADY_EXISTS`, but it looks like I misunderstood, I'll turn off this pr.